tests/data/use-cases/db-add-guardian.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { type AddGuardianRepository, type Encrypter } from '../../../src/data/protocols'
+import { DbAddGuardian } from '../../../src/data/use-cases'
```

Comment:
Remove the "../../../src/" from the imports, because the `src` folder is already set as the root of the project in typescript